CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 9.1.0-alpha.1 - February 28, 2020
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v9.0.0...android-v9.1.0-alpha.1) since [Mapbox Maps SDK for Android v9.0.0](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.0.0):
+### Features
+ - Introduce `in` expression for testing whether an item exists in an array or a substring exists in a string. ([#171](https://github.com/mapbox/mapbox-gl-native-android/pull/171))
```

Comment:
Nit: Previously we don't have brackets around the PR's number and link.

Reply:
Ah yeah - I added it intentionally because I think it’s better and makes the changelog clearer, but I didn’t say anything about it. Good eyes 😉